policy/modules/services/dbus.te

```diff
@@ -86,6 +86,7 @@ read_lnk_files_pattern(system_dbusd_t, dbusd_etc_t, dbusd_etc_t)
 manage_dirs_pattern(system_dbusd_t, system_dbusd_tmp_t, system_dbusd_tmp_t)
```

Comment:
Commit message: can you elaborate more detail which issues this fix fixes?

Reply:
I've changed the commit and added more informative description.